scripts/verbose_converter/src/benchdnn_generator.py

```diff
@@ -418,6 +418,43 @@ class BinaryConverter(AlgorithmMixin, MultiSourceMixin, Converter):
     @property
     def shapes(self):
         return self.entry.shapes.split(" ", 1)[0]
+    
+    @property
+    def dts(self):
+        src_dts: List[str] = []
+        other_dts: Dict[str, str] = {}
+        # limited the printed src mds to src0 and src1
+        src_ctr = 0
+        for md in self.entry.mds:
+            dt = md.data_type
+            if md.arg == "src":
+                if src_ctr < 2:
+                    src_dts.append(dt)
+                    src_ctr += 1
+            elif dt not in "undef":
+                other_dts[md.arg[0]] = dt
+        sdt_flags = "--sdt=" + ":".join(src_dts)
+        other_dt_flags = " ".join(f"--{k}dt={v}" for k, v in other_dts.items())
+        return f"{sdt_flags} {other_dt_flags}".strip()
+
+    @property
+    def tags(self):
+        src_tags: List[str] = []
+        other_tags: Dict[str, str] = {}
+        # limited the printed src mds to src0 and src1
+        src_ctr = 0
+        for md in self.entry.mds:
+            if md.arg == "src":
+                if src_ctr < 2:
+                    src_tags.append(maybe_make_any_tag(md))
+                    src_ctr += 1
+            elif md.tag:
+                other_tags[md.arg[0]] = maybe_make_any_tag(md)
+        stag_flags = "--stag=" + ":".join(src_tags)
+        other_tag_flags = " ".join(
+            f"--{k}tag={v}" for k, v in other_tags.items()
+        )
+        return f"{stag_flags} {other_tag_flags}".strip()
```

Comment:
I think since this is specialized for binary, we can simplify this quite a bit
```suggestion
    @property
    def dts(self):
        src0, src1, *_, dst = self.entry.mds
        return f"--sdt={src0.data_type}:{src1.data_type} {dst.data_type}"

    @property
    def tags(self):
        src0, src1, *_, dst = self.entry.mds
        src0_tag = maybe_make_any_tag(src0)
        src1_tag = maybe_make_any_tag(src1)
        dst_tag = maybe_make_any_tag(dst)
        return f"--stag={src0_tag}:{src1_tag} --dtag={dst_tag}"
```

It should make things a bit simpler for my request in #3229, too.

Reply:
Makes sense. I have updated the methods accordingly.